scraping/heavenly_scraper.py

```diff
@@ -0,0 +1,58 @@
+def ski_scraper(url):
```

Comment:
It seems everything in this directory can be removed. Project code should be in an app.

Reply:
Ah, this is my test code directory where I figure out the scraping details. It's in a confusing place and the name is very similar to my app, but maybe I can change the folder name. Maybe something like "development"?

Reply:
I would move this to a separate branch. Experimental code of this type shouldn't be on the master branch if you're maintaining good repository/project hygiene.

Reply:
Ok, that makes sense. I wanted to save them for reference, so I didn't want to put them into the .gitignore file. My guess is that I create the new branch, push it, remove this folder from master, and always keep the branches separate?